model/Core/Datatypes/MediaType.md

```diff
@@ -4,14 +4,12 @@ SPDX-License-Identifier: Community-Spec-1.0
 
 ## Summary
 
-Standardized way of indicating the type of content of an Element. A String constrained to the RFC 6838 specification.
+Standardized way of indicating the type of content of an Element. A String constrained to the RFC 2046 specification.
 
 ## Description
 
-A MediaType is a string constrained to the RFC 6838 specification, and includes
-`type-name`, `subtype-name` and optional `parameters`.
-It provides a standardized way of indicating the type of content of an Element.
-A list of all possible media types is available at https://www.iana.org/assignments/media-types/media-types.xhtml.
```

Comment:
Any reason why the list of possible media types is not being carried forward?    Looks like useful information to me.

Reply:
I just copied over the previous commit which refers to RFC 2046 - see issue #569 for the reason for this

Reply:
I understand the reversion, based on the issue history.    I just think that the phrase:  "A list of all possible media types is available at https://www.iana.org/assignments/media-types/media-types.xhtml."  is useful to add.  However, I'm fine with it being added later.

Reply:
Got it - I thought your comment was referring to a different line.  I'll add that in.